MapboxCoreNavigation/BundleAdditions.swift

```diff
@@ -28,4 +28,8 @@ extension Bundle {
             return object(forInfoDictionaryKey: "NSLocationAlwaysUsageDescription") as? String
         }
     }
+    
+    class var mapboxCoreNavigation: Bundle {
+        get { return Bundle(for: self) }
```

Comment:
`mapboxCoreNavigation` makes sense. `Bundle(for: self)` should work even though it takes a class but we're inside an extension. How about the root of core navigation, which is RouteController? `Bundle(for: RouteController.self)`